tests/integ/sagemaker/jumpstart/utils.py

```diff
@@ -53,23 +53,18 @@ def get_sm_session() -> Session:
     return Session(boto_session=boto3.Session(region_name=JUMPSTART_DEFAULT_REGION_NAME))
 
 
-# def get_sm_session_with_override() -> Session:
-#     # [TODO]: Remove service endpoint override before GA
-#     # boto3.set_stream_logger(name='botocore', level=logging.DEBUG)
-#     boto_session = boto3.Session(region_name="us-west-2")
-#     sagemaker = boto3.client(
-#         service_name="sagemaker-internal",
-#         endpoint_url="https://sagemaker.beta.us-west-2.ml-platform.aws.a2z.com",
-#     )
-#     sagemaker_runtime = boto3.client(
-#         service_name="runtime.maeve",
-#         endpoint_url="https://maeveruntime.beta.us-west-2.ml-platform.aws.a2z.com",
-#     )
-#     return Session(
-#         boto_session=boto_session,
-#         sagemaker_client=sagemaker,
-#         sagemaker_runtime_client=sagemaker_runtime,
-#     )
+def get_sm_session_with_override() -> Session:
+    # [TODO]: Remove service endpoint override before GA
+    # boto3.set_stream_logger(name='botocore', level=logging.DEBUG)
+    boto_session = boto3.Session(region_name="us-west-2")
+    sagemaker = boto3.client(
+        service_name="sagemaker",
+        endpoint_url="https://sagemaker.gamma.us-west-2.ml-platform.aws.a2z.com",
```

Comment:
Did you merge this to `main`? Block the release if that's the case.

Btw this code should never have been pushed to the public repo...